api/app/stations.py

```diff
@@ -42,6 +42,7 @@ def _get_stations_by_codes_local(station_codes: List[int]) -> List[WeatherStatio
 async def get_stations_by_codes(station_codes: List[int]) -> List[WeatherStation]:
     """ Get a list of stations by code, from WFWX Fireweather API. """
     if wildfire_one.use_wfwx():
```

Comment:
@andrea-williams oh - another thing. The 90th percentile calculator is going to have to keep on pointing at the pre-prepared geojson.